pydarn/plotting/grid.py

```diff
@@ -0,0 +1,261 @@
+                default: 0
+            start_time: datetime.datetime
+                datetime object as the start time of the record to plot
+                if none then record will be used
+                default: none
+            time_delta: int
+                How close the start_time has to be start_time of the record
+                in minutes
+                default: 1
+            ax: matplotlib.pyplot axis
+                Pre-defined axis object to pass in, must currently
+                be polar projection
+                Default: Generates a polar projection for the user
+                with MLT/latitude labels
+            parameter: str
+                Key name indicating which parameter to plot.
+                Default: vel (Velocity). Alternatives: 'pwr', 'wdt'
+            lowlat: int
+                Lower AACGM latitude boundary for the polar plot
+                Default: 50
+            fov: bool
+                Set to false to not plot the outline of the FOV
+                Default: True
+            cmap: matplotlib.cm
+                matplotlib colour map
+                https://matplotlib.org/tutorials/colors/colormaps.html
+                Default: Official pyDARN colour map for given parameter
+            zmin: int
+                The minimum parameter value for coloring
+                Default: {'pwr': [0], 'vel': [0], 'wdt': [0]}
+            zmax: int
+                The maximum parameter value for  coloring
+                Default: {'pwr': [50], 'vel': [1000], 'wdt': [250]}
+            colorbar: bool
+                Draw a colourbar if True
+                Default: True
+            colorbar_label: str
+                The label that appears next to the colour bar.
+                Requires colorbar to be true
+                Default: ''
+            title: str
+                Adds a title to the plot. If no title is specified,
+                one will be provided
+                Default: ''
+            len_factor: float
+                Normalisation factor for the vectors, to control size on plot
+                Larger number means smaller vectors on plot
+                Default: 150.0
+            ref_vector: int
+                Velocity value to be used for the reference vector, in m/s
+                Default: 300
+
+        See Also
+        --------
+        plot_fov - plots the field of view found in fan.py
+
+        Returns
+        -----------
+        if parameter is vector.vel.median:
+        thetas - list of data point longitudes in polar coordinates shifted
+        end_thetas - list of vector tails theta
+        rs - list of data point latitudes
+        end_rs - list of vector tails radius (latitude)
+        data - Line of Sight points of parameter chosen
+        else:
+        thetas - list of data point longitudes in polar coordinates shifted
+        rs - list of data point latitudes
+        data - gridded vector points of the parameter chosen
```

Comment:
```suggestion
        data - List of data magnitudes plotted, for parameter chosen
```
If it's not velocity then it's not a vector.